less/release-9.less

```diff
@@ -0,0 +1,351 @@
+    }
+  }
+
+  .jumbotron-right-content {
+    font-family: 'Exo', sans-serif;
+    font-size: 130px;
+    @media (min-width: @screen-sm-min) {
+      font-size: 250px;
+    }
+    @media (min-width: @screen-md-min) {
+      font-size: 250px;
+    }
+    @media (min-width: @screen-lg-min) {
+      font-size: 250px;
+    }
+    line-height: 1;
+    font-weight: 400;
+  }
+  ul.buttons {
+    li {
+      width:100%;
+      @media (min-width: @screen-md-min) {
+        width:49%;
+      }
+    }
+  }
+  .btn {
+    width:100%;
+    @media (max-width: @screen-sm-max) {
+      margin-bottom:15px;
+    }
+  }
+}
+  
+
+.features-and-benefits {
+    padding:30px 0 60px 0;
+
+    .feature-box {
+        background-color: #eeeded;
+        margin: 10px;
+        padding: 0 40px;
+        border-top-left-radius: 30px;
+        border-top-right-radius: 30px;
+        border-bottom-left-radius: 5px;
+        border-bottom-right-radius: 5px;
+        -webkit-box-shadow: -20px 25px 23px 0px rgba(156,156,156,1);
+        -moz-box-shadow: -20px 25px 23px 0px rgba(156,156,156,1);
+        box-shadow: -20px 25px 23px 0px rgba(156,156,156,1);
+
+        h3 {
+            padding-top: 70px;
+        }
+
+        p {
+            padding-bottom: 30px;
+        }
+
+        hr {
+            max-width: 150px;
+            border-top: 5px solid @brand-primary;
+            margin-left: 0;
+        }
+
+    }
+
+    .polygons {
+        background: @brand-secondary;
+        height: 75px;
+        position: relative;
+        width: 70px;
+
+    }
+
+    .polygons:after {
+        border-top: 10px solid @brand-secondary;
+        border-left: 35px solid transparent;
+        border-right: 35px solid transparent;
+        content: "";
+        height: 0;
+        left: 0;
+        position: absolute;
+        top: 75px;
+        width: 0;
+    }
+
+    .polygons-right {
+        float: right;
+        margin-top: -10px;
+    }
+
+}
+
+.jakarta-ee-hr {
+    width: 150px;
+    border-top: 5px solid @brand-primary;
+    margin-left: 0;
+}
+
+
+.font-numbers {
+  font-size: 100px;
+}
+
+.btn-watch-video {
+  &:hover {
+    background-color: @brand-primary;
+    border-color: @brand-primary;
+  }
+}
+
+.white-paper-bg {
+
+  //* background *//
+  background-image: url("images/release/release-9-download-mobile-bg.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  overflow-x: hidden;
+  overflow-y: visible;
+
+  @media (min-width: @screen-lg-min) { // 1200px
+    background-position: top;
+  }
+
+  @media (min-width: @screen-sm-min) {  // 768px
+    background-image: url("images/release/jakarta9-image.png");
+    background-position: left;
+    padding-top: 200px;
+    padding-bottom: 200px;
+  }
+  //* background *//
+
+  // padding-left: 0;
+  // padding-right: 0;
+  .btn-blue {
+    background-color: @white;
+    color: @brand-secondary;
+  }
+  .btn-blue:hover {
+    background-color: @brand-secondary;
+    color: @white;
+  }
+}
+
+.text-white {
```

Comment:
This class already exist:
https://github.com/EclipseFdn/solstice-assets/blob/master/less/_components/classes-colors.less#L135